libs/platform/src/lib/form/multi-combobox/multi-combobox-mobile/multi-combobox/multi-combobox-mobile.component.html

```diff
@@ -1,16 +1,17 @@
 <ng-template let-dialog let-dialogConfig="dialogConfig" #dialogTemplate>
     <fd-dialog [dialogConfig]="dialogConfig" [dialogRef]="dialog">
-        <fd-dialog-header>
+        <fd-dialog-header [attr.aria-label]="mobileConfig.title">
```

Comment:
Can we use [DialogTitleDirective](https://github.com/SAP/fundamental-ngx/blob/57378b5d732ec3986b0c84dcf60629cb830f9397/libs/core/src/lib/dialog/directives/dialog-title.directive.ts#L14) instead?

Reply:
@dimamarksman , you mean using `fd-dialog-title` instead of `[attr.aria-label]`?

Reply:
@dimamarksman , could you please describe in more detail your suggested change?

Reply:
got it wrong, sorry